File: tests/fixtures/discrimination/javascript/out_of_corpus/fizzbuzz.js
function fizzbuzz(n) {
  var results = [];
  for (var i = 1; i <= n; i++) {
    if (i % 15 === 0) {
      results.push("FizzBuzz");
    } else if (i % 3 === 0) {
      results.push("Fizz");
    } else if (i % 5 === 0) {
      results.push("Buzz");
    } else {
      results.push(i);
    }
  }
  return results;
}

function isPrime(num) {
  if (num < 2) return false;
  for (var i = 2; i <= Math.sqrt(num); i++) {
    if (num % i === 0) return false;
  }
  return true;
}

module.exports = { fizzbuzz, isPrime };

File: tests/fixtures/discrimination/rust/out_of_corpus/fizzbuzz.rs
fn fizzbuzz(n: u32) -> String {
    if n % 15 == 0 {
        String::from("FizzBuzz")
    } else if n % 3 == 0 {
        String::from("Fizz")
    } else if n % 5 == 0 {
        String::from("Buzz")
    } else {
        n.to_string()
    }
}

fn run_fizzbuzz(limit: u32) {
    for i in 1..=limit {
        println!("{}", fizzbuzz(i));
    }
}

fn main() {
    run_fizzbuzz(20);
}
